tests/macfuse_smoke.rs

```diff
@@ -53,7 +53,7 @@ mod macfuse_tests {
     }
 
     #[test]
-    #[ignore] // it depends on privileged mode to pass through /dev/fuse
+    #[cfg(feature = "fuse-t")] // it depends on privileged mode to pass through /dev/fuse
```

Comment:
we use `--ignore` to test with privilege mode.

Reply:
This case only run macos, has no `--ignore` run.

Reply:
macfuse can not get privilege, becase it must click in `System Setting`.